homeassistant/components/nfandroidtv/__init__.py

```diff
@@ -1 +1,69 @@
-"""The nfandroidtv component."""
+"""The NFAndroidTV integration."""
+import logging
+
+from notifications_android_tv.notifications import ConnectError, Notifications
+
+from homeassistant.components.notify import DOMAIN as NOTIFY
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PLATFORM
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import discovery
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [NOTIFY]
+
+
+async def async_setup(hass: HomeAssistant, config):
+    """Set up the NFAndroidTV component."""
+    hass.data.setdefault(DOMAIN, {})
+    # Iterate all entries for notify to only get nfandroidtv
+    if NOTIFY in config:
+        for entry in config[NOTIFY]:
+            if entry[CONF_PLATFORM] == DOMAIN:
+                hass.async_create_task(
+                    hass.config_entries.flow.async_init(
```

Comment:
Don't import the config here. Do it from the platform setup. Platform config can take different forms in YAML so it's not safe to import that from here.